astropy/time/tests/test_mask.py

```diff
@@ -15,6 +16,10 @@
                                  atol=2. ** -52 * 24 * 3600)  # 20 ps atol
 is_masked = np.ma.is_masked
 
+# This is r"can't set attribute '{0}'" for Python 3.10, but not 3.9.
```

Comment:
Shouldn't it say
```suggestion
# This is r"can't set attribute '{0}'" for Python 3.10, but not 3.11.
```
See also https://github.com/astropy/astropy/pull/13614#discussion_r968602390

Reply:
No, it should not – see also https://github.com/astropy/astropy/pull/13614#discussion_r968611756.
Can try to make it clearer that it is about the changes from 3.9 to 3.10, or add an extra `PYTHON_LT_3_10` case right away (otherwise I would have just waited until 3.9 support is dropped to switch to the more specific version).